Python: Added some missing config options (#10546)
getsentry/sentry-docs: docs/platforms/python/configuration/options.mdx

</ConfigKey>

<ConfigKey name="dist">

> The distribution of the application. Distributions are used to disambiguate build or deployment variants of the same release of an application. The dist can be for example a build number.

Reviewer: ```suggestion
Application distribution. This can be used to disambiguate build or deployment variants that are part of the same release. (For example, the dist can be a build number.)
```

</ConfigKey>

<ConfigKey name="project_root">

The full path to the root directory of your application. Defaults to `os.getcwd()`.

> The `project_root` is used to mark frames in a stack trace either as being in your application or outside of the application.

Reviewer: ```suggestion
The `project_root` is used to mark frames in a stack trace as either being in or outside of your application.
```
